domains/certificates/Query.API/API.AppTests/HealthTests.cs

```diff
@@ -1,17 +1,22 @@
 using System.Net;
 using System.Threading.Tasks;
 using API.AppTests.Infrastructure;
+using API.IntegrationTest.Infrastructure;
 using Xunit;
 
 namespace API.AppTests;
 
-public class HealthTests : IClassFixture<QueryApiWebApplicationFactory>
+public class HealthTests : IClassFixture<QueryApiWebApplicationFactory>, IClassFixture<RabbitMqContainer>
```

Comment:
Now that we a way of getting the application in an unhealthy state (by not having a RabbitMQ container running), should we have a test that checks that the application becomes unhealthy?

Reply:
Good point, i do have some troubles getting the health check to act as i want it to act. :) Maybe we can have a chat about it?

Reply:
Sure

Reply:
As agreed an unhealthtests class has been created